project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementLookupService.java

```diff
@@ -68,6 +68,14 @@ public List<NGSMeasurement> queryNGSMeasurementsBySampleIds(String termFilter,
     return new ArrayList<>(termList);
   }
 
+  public long countNGSMeasurementsBySampleIds(Collection<SampleId> sampleIds) {
```

Comment:
This seems not to be used by the project view, instead each experiment is used to count sample ids - might this be faster? also: why was it changed in this PR?

Reply:
ah, nevermind